### PR TITLE
rollback is not always exceptional

### PR DESCRIPTION
**Pull request by jonhawkes**
#build

### CODE REVIEW

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build 
#run-libby-bot

File: dev/com.ibm.ws.transaction.context_test/test/src/com/ibm/ws/transaction/context/internal/TransactionContextTest.java [hunk 1]
```diff
@@ -0,0 +1,268 @@
+package com.ibm.ws.transaction.context.internal;
```
Comment (tevans78) [inline dev/com.ibm.ws.transaction.context_test/test/src/com/ibm/ws/transaction/context/internal/TransactionContextTest.java]: Please add the EPL copyright header

**Comment by jonhawkes** [comment]
#build 
#run-libby-bot

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
#build

**Comment by jonhawkes** [comment]
fixes #11696

**Pull request merged**